chart/templates/server-ide-configmap.yaml

```diff
@@ -104,6 +104,34 @@ options:
 
 defaultIde: "code"
 defaultDesktopIde: "code-desktop"
+
+clients:
+  vscode:
+    defaultDesktopIDE: "code-desktop"
+    desktopIDEs: ["code-desktop"]
+    installationSteps: [
+      "Click <b>Allow</b> in the dialog by your browser.",
+      "If you don't see the dialog, make sure you have <a target='_blank' class='gp-link' href='https://code.visualstudio.com/download'>VS Code</a> installed on your machine.",
+      "After that, click the <b>${OPEN_LINK_LABEL}</b> button below."
+    ]
+  vscode-insiders:
+    defaultDesktopIDE: "code-desktop-insiders"
+    desktopIDEs: ["code-desktop-insiders"]
+    installationSteps: [
+      "Click <b>Allow</b> in the dialog by your browser.",
+      "If you don't see the dialog, make sure you have <a target='_blank' class='gp-link' href='https://code.visualstudio.com/insiders'>VS Code Insiders</a> installed on your machine.",
+      "After that, click the <b>${OPEN_LINK_LABEL}</b> button below."
+    ]
+  jetbrains-gateway:
+    defaultDesktopIDE: "intellij"
+    desktopIDEs: ["intellij", "goland", "pycharm", "phpstorm"]
+    installationSteps: [
+      "Click <b>Allow</b> in the dialog by your browser.",
+      "If you don't see the dialog, make sure you have JetBrains Gateway with Gitpod plugin installed on your machine:",
+      " - To install JetBrains Gateway, use <a target='_blank' class='gp-link' href='https://www.jetbrains.com/toolbox-app/'>JetBrains Toolbox App</a>.",
+      " - <a target='_blank' class='gp-link' href='https://plugins.jetbrains.com/plugin/download?rel=true&updateId=154074'>Download the plugin</a> and <a target='_blank' class='gp-link' href='https://www.jetbrains.com/help/idea/managing-plugins.html#install_plugin_from_disk'>install it from the disk</a>.",
```

Comment:
**question:** Do we need this separate kind of intructions? Gateway users can probably familiarize themselves with how to install plugins. 🤷 

Similar to other editors like VS Code, we could re-use the same text for instructions and simply append the required software which is the plugin in this case. What do you think? Do we need to be more explicit in these instructions? 💭 

> If you don't see the dialog, make sure pop-ups are allowed in this browser, you have <a href="https://www.jetbrains.com/toolbox-app/">JetBrains Gateway</a> and the <a href="https://plugins.jetbrains.com/plugin/download?rel=true&updateId=154074">Gitpod Plugin</a> installed, and then try clicking **Open IntelliJ on Desktop** below.

Reply:
I will drop about pop-ups since they are not relevant for desktop apps, but the rest I like. 👍

Reply:
> If you don't see an open dialog by the browser, make sure you have <a href="https://www.jetbrains.com/toolbox-app/">JetBrains Gateway</a> and <a href="https://plugins.jetbrains.com/plugin/download?rel=true&updateId=154074">Gitpod Plugin</a> installed on your machine, and then try clicking **Open IntelliJ on Desktop** below.

I added:
- `an open dialog by the browser` - I don't think just jus `the dialog` is clear for a user 
- `installed on your machine` - to make clear that it should not be installed in the browser or in Gitpod, but on user machine

Sounds good?

Reply:
>  I don't think just jus the dialog is clear for a user

Completely agree. I was about to submit a comment about this.

>  to make clear that it should not be installed in the browser or in Gitpod, but on user machine

DEAL 🤝